src/main/java/com/paneedah/weaponlib/sound/RealisticSound.java

```diff
@@ -0,0 +1,50 @@
+package com.paneedah.weaponlib.sound;
+
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+
+public class RealisticSound {
+
+    private final SoundEvent sound;
+    private final float volumeIn;
+    private final float pitchIn;
+
+    private static final double MAX_DISTANCE = 400D;
+    private static final double MAX_DISTANCE_SILENCER = 100D;
```

Comment:
the obsession will never be stopped
```suggestion
    private static final double MAX_DISTANCE = 400D;
    private static final double MAX_DISTANCE_SILENCER = 100D;

    private final float volumeIn;
    private final float pitchIn;

    private final SoundEvent sound;
```